### PR TITLE
[BB-2801] Update default Open EdX release to Juniper

### PR DESCRIPTION
**Pull request by 0x29a**
Updated default settings to use Juniper release and also fixed an issue with the forum heartbeat on Juniper.

**JIRA tickets**:
- [BB-2801](https://tasks.opencraft.com/browse/BB-2801)

**Dependencies**:
This PR depends on:
2. https://github.com/open-craft/ansible-secrets/pull/196

**Testing instructions**:
1. Deploy the changes and verify that a working Juniper instance is spawned.

**Author notes and concerns**:

**Reviewers**
- [ ] @lgp171188

### CODE REVIEW

File: opencraft/settings.py [hunk 1]
```diff
@@ -430,7 +430,7 @@
 STABLE_CONFIGURATION_REPO_URL = env(
     'STABLE_CONFIGURATION_REPO_URL', default=DEFAULT_CONFIGURATION_REPO_URL
 )
-STABLE_CONFIGURATION_VERSION = env('STABLE_CONFIGURATION_VERSION', default='opencraft-release/ironwood.2')
+STABLE_CONFIGURATION_VERSION = env('STABLE_CONFIGURATION_VERSION', default='opencraft-release/juniper.2')
```
Comment (lgp171188) [inline opencraft/settings.py:433]: @demid, `DEFAULT_OPENEDX_RELEASE` has to be updated as well. We also need to create a PR against the ansible secrets repository where the configuration to set these environment variables are stored.

File: instance/tests/integration/factories/instance.py [hunk 2]
```diff
@@ -79,8 +79,8 @@ def create(cls, *args, **kwargs):  # pylint: disable=arguments-differ
     # or a release candidate tag will work.  We point both the edx-platform and the configuration
     # versions to the branch "integration" in our own forks.  These branches are based on the
     # corresponding openedx_release versions from upstream, but can contain custom modifications.
-    openedx_release = 'open-release/ironwood.2'
+    openedx_release = 'open-release/juniper.2'
     configuration_source_repo_url = 'https://github.com/open-craft/configuration.git'
-    configuration_version = 'integration-ironwood'
+    configuration_version = 'integration-juniper'
```
Comment (lgp171188) [inline instance/tests/integration/factories/instance.py:84]: @0x29a, you can drop this change from this PR if this is causing CI check failures. We can investigate and fix the issues and create a separate PR.

Comment (0x29a) [reply instance/tests/integration/factories/instance.py:84]: @lgp171188, ah, okay, I'll do this. Thanks.

Comment (0x29a) [reply instance/tests/integration/factories/instance.py]: @lgp171188 done.

**Pull request merged**